### DISCUSSION TITLE
제패토월드 템플릿 import에러

### DISCUSSION THREAD

**Discussion by vencerr** [naverz/zepeto-studio-kor#1171] [5. Q&A]
![캡처](https://user-images.githubusercontent.com/111266081/184584169-80c24cf6-d43b-4837-9d6f-ca02da416b83.PNG)

안녕하세요.
제페토 월드템플릿을 다운하고 import했는데
이런 에러가 뜨네요..
어떻게 해결하나요ㅠ?

**Comment by zlmrzlo** [comment]
씬을 다운로드 받은 씬으로 변경하면 될 것 같아요.
